### ISSUE TITLE
sitecore-jss-react-native depends on unsupported react-native-svg-uri

### ISSUE THREAD

**Issue by warp** [Sitecore/jss#1078]
## Description
[react-native-svg-uri](https://www.npmjs.com/package/react-native-svg-uri) was last published 5 years ago and uses deprecated react features.

## Expected behavior
sitecore-jss-react-native should work with current react-native versions

## Steps To Reproduce
`npx create-sitecore-jss react-native` and attempt to run with `yarn start-ios`

## Possible Fix
Newer versions of react-native-svg now have similar functionality built in so it would be preferable to use that instead.

## Your Environment
* Sitecore Version: All
* JSS Version: 21.0.0-canary.111
* Browser Name and version: N/A
* Operating System and version (desktop or mobile): Mobile
* Link to your project (if available):

## Screenshots
![image](https://user-images.githubusercontent.com/10120/176098789-9e41c11f-ddf4-45c6-8778-3c1c11b90a1f.png)

**Comment by art-alexeyenko** [comment]
Thank you for reporting it, and especially for the PR request follow up!
We are currently planning a more expanded update and refresh to React Native JSS (in the scope of the next major release). React native svg should be updated along with some other libraries as a part of it, so we will post in this thread once that is done.

**Comment by art-alexeyenko** [comment]
I should update this. Unfortunately the react-native work has been postponed by other features and tasks.
Currently no ETA on when this might be implemented, as react-native remains in "experimental" support for the time being.
